src/test/kotlin/game/racing/domain/RacingManagerTest.kt

```diff
@@ -1,21 +1,39 @@
 package game.racing.domain
 
 import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
 import org.junit.jupiter.params.ParameterizedTest
-import org.junit.jupiter.params.provider.ValueSource
+import org.junit.jupiter.params.provider.MethodSource
 
 internal class RacingManagerTest {
     @ParameterizedTest
-    @ValueSource(strings = ["WIN,TEST2,TEST3", "TEST1,TEST2,WIN,TEST4", "TEST1,WIN,TEST3,TEST4,TEST5"])
-    fun `우승자를 반환한다`(carNames: String) {
-        val cars = carNames.split(",").toTypedArray().map { Car(it) }
+    @MethodSource("provideCarList")
+    fun `우승자를 반환한다`(cars: List<Car>) {
+        // when
         val racingManager = RacingManager(cars)
-        cars.forEach {
-            if (it.name == "WIN") {
-                it.moveOrStayBySpeed(Car.MIN_MOVE_THRESHOLD)
-            }
+        cars.filter { it.name == "WIN" }.forEach {
+            it.moveOrStayBySpeed(Car.MIN_MOVE_THRESHOLD)
         }
 
+        // then
         assertThat(racingManager.determineWinner()).allMatch { it.position == Car.MIN_POSITION + 1 }.allMatch { it.name == "WIN" }
     }
+
+    @Test
+    fun `자동차가 움직인다`() {
+        val cars = arrayOf(Car("TEST1"), Car("TEST2"), Car("TEST3"))
+        val racingManager = RacingManager(cars.toList())
```

Comment:
처음부터 list를 생성하면 list로 변환할 필요가 없을 것 같아요. 😉
```suggestion
        val cars = listOf(Car("TEST1"), Car("TEST2"), Car("TEST3"))
        val racingManager = RacingManager(cars)
```